ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/snapshot/services/SnapshotServiceImpl.java

```diff
@@ -47,4 +55,8 @@ private void createSnapshotForTreeNodes(List<LdesFragment> treeNodesForSnapshot,
 		snapshotRepository.saveSnapShot(snapshot);
 	}
 
+	private Optional<Snapshot> retrieveLastSnapshot() {
+		return snapshotRepository.getLastSnapshot();
```

Comment:
Why not just take line 33-45 entirely in this function? Since for now it looks like a fix for an unclear name of a repo method :)